### ISSUE TITLE
Xcode 10/Cocoapods: No such module 'Moya'

### ISSUE THREAD

**Issue by nmtitov** [Moya/Moya#1741]
I can't get `Archive` working for my Cocoapods workspace and Moya (tried both 11 and 12).
I can run my project on the Simulator or a real device without any problems. However, `Archive` doesn't work.

```swift
import Moya
```

The message is:

```
No such module 'Moya'
```

Versions used:

```
Xcode Version 10.0 (10A255)
pod --version 1.5.3
PODS:
  - Moya (12.0.0-beta.1):
    - Moya/Core (= 12.0.0-beta.1)
  - Moya/Core (12.0.0-beta.1):
    - Alamofire (~> 4.1)
    - Result (~> 4.0)
  - Moya/ReactiveSwift (12.0.0-beta.1):
    - Moya/Core
    - ReactiveSwift (~> 4.0)
```

See screenshots attached below.

<img width="315" alt="screen shot 2018-10-10 at 14 22 37" src="https://user-images.githubusercontent.com/193307/46736256-decef300-cc98-11e8-8a5d-610640e69a19.png">
<img width="261" alt="screen shot 2018-10-10 at 14 22 20" src="https://user-images.githubusercontent.com/193307/46736259-e1c9e380-cc98-11e8-9d41-314d12e21804.png">
<img width="665" alt="screen shot 2018-10-10 at 14 22 14" src="https://user-images.githubusercontent.com/193307/46736262-e4c4d400-cc98-11e8-9334-ed0db5d3bebc.png">
<img width="673" alt="screen shot 2018-10-10 at 14 22 02" src="https://user-images.githubusercontent.com/193307/46736269-e8585b00-cc98-11e8-96e7-3acc27ebd559.png">

Same for Moya 11.

Any help is greatly appreciated! Thanks!

Upd:

`pod install`
```
Analyzing dependencies
Downloading dependencies
Using Alamofire (4.7.3)
Using Bolts (1.9.0)
Using CMPopTipView (2.3.2)
Using FBSDKCoreKit (4.37.0)
Using FBSDKLoginKit (4.37.0)
Using FacebookCore (0.4.0)
Using FacebookLogin (0.4.0)
Using Helpshift (7.1.1)
Using Moya (11.0.2)
Using NSLogger (1.9.0)
Using OneSignal (2.8.9)
Using ReactiveCocoa (7.2.0)
Using ReactiveSwift (3.1.0)
Using Result (3.2.4)
Using Rollbar (1.0.0)
Using SDWebImage (4.4.2)
Using SnapKit (4.0.1)
Generating Pods project
Integrating client project
Sending stats
Pod installation complete! There are 12 dependencies from the Podfile and 17 total pods installed.
```

`cat Podfile.lock `
```
PODS:
  - Alamofire (4.7.3)
  - Bolts (1.9.0):
    - Bolts/AppLinks (= 1.9.0)
    - Bolts/Tasks (= 1.9.0)
  - Bolts/AppLinks (1.9.0):
    - Bolts/Tasks
  - Bolts/Tasks (1.9.0)
  - CMPopTipView (2.3.2)
  - FacebookCore (0.4.0):
    - Bolts (~> 1.9)
    - FBSDKCoreKit (~> 4.36)
  - FacebookLogin (0.4.0):
    - Bolts (~> 1.9)
    - FacebookCore (~> 0.4)
    - FBSDKCoreKit (~> 4.36)
    - FBSDKLoginKit (~> 4.36)
  - FBSDKCoreKit (4.37.0):
    - Bolts (~> 1.7)
  - FBSDKLoginKit (4.37.0):
    - FBSDKCoreKit
  - Helpshift (7.1.1)
  - Moya (11.0.2):
    - Moya/Core (= 11.0.2)
  - Moya/Core (11.0.2):
    - Alamofire (~> 4.1)
    - Result (~> 3.0)
  - Moya/ReactiveSwift (11.0.2):
    - Moya/Core
    - ReactiveSwift (~> 3.0)
  - NSLogger/ObjC (1.9.0)
  - NSLogger/Swift (1.9.0):
    - NSLogger/ObjC
  - OneSignal (2.8.9)
  - ReactiveCocoa (7.2.0):
    - ReactiveSwift (~> 3.1)
  - ReactiveSwift (3.1.0):
    - Result (~> 3.2)
  - Result (3.2.4)
  - Rollbar (1.0.0)
  - SDWebImage (4.4.2):
    - SDWebImage/Core (= 4.4.2)
  - SDWebImage/Core (4.4.2)
  - SnapKit (4.0.1)

DEPENDENCIES:
  - CMPopTipView (~> 2.3.2)
  - FacebookCore
  - FacebookLogin
  - Helpshift (= 7.1.1)
  - Moya
  - Moya/ReactiveSwift
  - NSLogger/Swift
  - OneSignal (from `https://github.com/OneSignal/OneSignal-iOS-SDK.git`, branch `beta`)
  - ReactiveCocoa
  - Rollbar (~> 1.0.0)
  - SDWebImage
  - SnapKit (~> 4.0.0)

SPEC REPOS:
  https://github.com/cocoapods/specs.git:
    - Alamofire
    - Bolts
    - CMPopTipView
    - FacebookCore
    - FacebookLogin
    - FBSDKCoreKit
    - FBSDKLoginKit
    - Helpshift
    - Moya
    - NSLogger
    - ReactiveCocoa
    - ReactiveSwift
    - Result
    - Rollbar
    - SDWebImage
    - SnapKit

EXTERNAL SOURCES:
  OneSignal:
    :branch: beta
    :git: https://github.com/OneSignal/OneSignal-iOS-SDK.git

CHECKOUT OPTIONS:
  OneSignal:
    :commit: 3ab8c706a0982252b4938a8f700896f44cae4a48
    :git: https://github.com/OneSignal/OneSignal-iOS-SDK.git

SPEC CHECKSUMS:
  Alamofire: c7287b6e5d7da964a70935e5db17046b7fde6568
  Bolts: ac6567323eac61e203f6a9763667d0f711be34c8
  CMPopTipView: 2d56ef53e8e739c4e63ad97a5a88b97712b46c7b
  FacebookCore: 770e57cfe250a4de4458b682191c0ff062dee247
  FacebookLogin: eb0f2251aca835a94d70a5f7e580edc46d1e1603
  FBSDKCoreKit: fe5f3474499a81963e11e3f3a5c753d0a95ca2b4
  FBSDKLoginKit: 2f7249686d1e30ce8a5ef5400eedf50b3e3df332
  Helpshift: 3a0dbb40757bb01f033f15450f7add78e52c735f
  Moya: a725035953bc1c0eb1be505ab903984501d82440
  NSLogger: 9f8f5ab017eb34d1a138f4905dc7fb8e9a0e3186
  OneSignal: 0fb3522620d392195ed8e90e4e96717c6085df45
  ReactiveCocoa: 476dae7ca790f40d30d186a1bd7ebae4290395d5
  ReactiveSwift: 5b26d2e988fe0eed2daf48c4054d1de74db50184
  Result: d2d07204ce72856f1fd9130bbe42c35a7b0fea10
  Rollbar: b1ad0f5159db93d056ac0f33291faef3367b761d
  SDWebImage: 624d6e296c69b244bcede364c72ae0430ac14681
  SnapKit: 0de968a9fec17499afa29683b05d0c775b6d1c29

PODFILE CHECKSUM: a6718dbb69938b8094b22a1078fa11c6e09edd23

COCOAPODS: 1.5.3
```

**Comment by nmtitov** [comment]
Same for the latest stable Moya version 11.

**Comment by initGalen** [comment]
me too, XCTest can not use

**Comment by SD10** [comment]
This should be fixed by Moya version 12.0. Please upgrade and feel free to re-open if the problem persists. Thank you for your patience 🙏

**Comment by Tawfikk** [comment]
@SD10 un Moya 12.0.1 such error!
